tests/unit_tests/test_util.py

```diff
@@ -51,3 +51,23 @@ def test_constructor_suffix_vector(self):
         getter = SuffixFilter('barz', ['.b', '.c'])
         result = getter(artefact_store={'barz': [Path('bar.a'), Path('bar.b'), Path('bar.c')]})
         assert result == [Path('bar.b'), Path('bar.c')]
+
+
+class Test_file_walk(object):
+
+    @pytest.fixture
+    def files(self, tmp_path):
+        f = tmp_path / 'foo/bar/foo.txt'
+        pbf = tmp_path / 'foo/bar/_prebuild/foo.txt'
+
+        pbf.parent.mkdir(parents=True)
+        f.touch()
+        pbf.touch()
+
+        return f, pbf
+
+    def test_ignore(self, files, tmp_path):
```

Comment:
I'm not sure this is doing the right thing. Doesn't `tmp_path` used here and `tmp_path` used by `files` generate different paths?

Reply:
Good question but no, it's set per test invocation. In this case, printing from both prints `/var/tmp/pytest-of-bblay/pytest-1356/test_ignore0` twice.